### PR TITLE
setup for models.py

### PR DESCRIPTION
**Pull request by KHafner**
# Pull Request Template

## Checklist:
- [x] I have written test cases for this change
- [x] I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story

### CODE REVIEW

**Comment by iLtc** [comment]
Should we have some test cases for the models?

**Review by jules203 — APPROVED** [review]
Looks good!

File: service/models.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    """
+
+    logger = logging.getLogger(__name__)
+    app = None
+
+    ##################################################
+    # Shopcart Table Schema
+    ##################################################
+    id = db.Column(db.Integer, primary_key=True)
+    uid = db.Column(db.Integer)
```
Comment (gauridhawan) [inline service/models.py:67]: Can we rename this to `user_id`?

Comment (KHafner) [reply service/models.py]: Sure, I just changed it

**Review by nonococoleo — APPROVED** [review]

**Comment by iLtc** [comment]
This PR has not passed all the tests. It requires `app` to work. Which is in #3. Let's wait until @nonococoleo finish #3.

**Pull request merged**